### DISCUSSION TITLE
Kata2

### DISCUSSION THREAD

**Discussion by EduardoLozadaQuiroz** [LaunchX-InnovaccionVirtual/CursoIntroPython#179] [Q&A] [answered]
Buenas noches compañeros, tengo una duda y es que no recuerdo en donde se va a subir la Kata2 ¿Alguien me puede ayudar con ello?
De ante mano muchas gracias y que tengan buena noche.

**Comment by OscarPedroza** [comment, marked as answer]
Hola buenas noches, puedes realizar la kata 2 con capturas de pantalla en un notebook o en un pdf y subirlos al repositorio con las demás katas, saludos.

**Reply by JosueVerAlar** [reply]
Como bien lo menciona @OscarPedroza , crea un notebook o un pdf con capturas de pantalla y subes ese archivo a tu repositorio, saludos!

**Reply by EduardoLozadaQuiroz** [reply]
Muchisimas gracias, de verdad por haberme ayudado con ello, que tengan buena noche.
